### PR TITLE
Request review of pack contributor

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold 

## Related Issues
fixes: https://github.com/demisto/etc/issues/25229

## Description
Added github action to request PR review of pack contributor user (will be defined in pack_metadata).

### CODE REVIEW

**Comment by IkaDemisto** [comment]
Example: https://github.com/demisto/content/pull/7941

File: .github/workflows/helper_scripts/request_contributor_review.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    for pack in modified_packs:
+        pack_metadata_path = os.path.join(PACKS_FULL_PATH, pack, PACK_METADATA)
+
+        if os.path.exists(pack_metadata_path):
+            with open(pack_metadata_path, 'r') as pack_metadata_file:
+                pack_metadata = json.load(pack_metadata_file)
+
+            if pack_metadata.get('support') != XSOAR_SUPPORT and PACK_METADATA_GITHUB_USER_FIELD in pack_metadata \
+                    and pack_metadata[PACK_METADATA_GITHUB_USER_FIELD]:
```
Comment (yaakovi) [inline .github/workflows/helper_scripts/request_contributor_review.py:98]:
```suggestion
            if pack_metadata.get('support') != XSOAR_SUPPORT and pack_metadata.get(PACK_METADATA_GITHUB_USER_FIELD):
```

File: .github/workflows/helper_scripts/request_contributor_review.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                pack_reviewers = pack_metadata[PACK_METADATA_GITHUB_USER_FIELD]
+                pack_reviewers = pack_reviewers if isinstance(pack_reviewers, list) else pack_reviewers.split(",")
+                github_users = [u.lower() for u in pack_reviewers]
```
Comment (yaakovi) [inline .github/workflows/helper_scripts/request_contributor_review.py:101]: users are case insensitive?

File: .github/workflows/helper_scripts/request_contributor_review.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        sys.exit(1)
+
+    pr_changed_data = response.json()
+    pr_files = [f.get('filename') for f in pr_changed_data]
+    modified_packs = {Path(p).parts[1] for p in pr_files if p.startswith(PACKS_FOLDER) and len(Path(p).parts) > 1}
+
+    return modified_packs
+
+
+def request_review_from_user(reviewers, pr_number, github_token=None, verify_ssl=True):
```
Comment (yaakovi) [inline .github/workflows/helper_scripts/request_contributor_review.py:70]: not sure we can ask external users for a review (outside demisto organization).
please test and switch to comment if not possible. (if you don't have a personal user - use mine: shaiy310)

File: .github/workflows/helper_scripts/request_contributor_review.py [hunk 4]
```diff
@@ -0,0 +1,142 @@
+import argparse
```
Comment (yaakovi) [inline .github/workflows/helper_scripts/request_contributor_review.py:1]: move script to `Utils` folder

File: .github/workflows/helper_scripts/request_contributor_review.py (hunk 2 above)
Comment (IkaDemisto) [reply .github/workflows/helper_scripts/request_contributor_review.py]: In gihtub api users are case insensitive for some reason.

File: .github/workflows/helper_scripts/request_contributor_review.py (hunk 3 above)
Comment (IkaDemisto) [reply .github/workflows/helper_scripts/request_contributor_review.py]: Changed to comment instead

**Comment by IkaDemisto** [comment]
@yaakovi done

**Pull request merged**